reflex/constants.py

```diff
@@ -69,9 +69,9 @@ def get_fnm_name() -> str | None:
 
 # Bun config.
 # The Bun version.
-BUN_VERSION = "0.7.0"
+BUN_VERSION = "0.7.3"
 # Min Bun Version
-MIN_BUN_VERSION = "0.7.0"
+MIN_BUN_VERSION = "0.7.3"
```

Comment:
I think we should leave the `MIN_BUN_VERSION` alone, since we _can_ run on 0.7.0, it just causes issues in certain configurations. The `BUN_VERSION` determines what will get installed by reflex when it is bootstrapping.

Reply:
Sounds good. Pushed a change with that feedback.